pandas/parser.pyx

```diff
@@ -1239,6 +1239,10 @@ class CParserError(ValueError):
     pass
 
 
+class EmptyDataError(ValueError):
+    pass
```

Comment:
Any objections to attempting to removing it?  A quick search suggests it may not be too painful.